### DISCUSSION TITLE
Confirmed to work on 2024 Telluride with "Kia Access"

### DISCUSSION THREAD

**Discussion by Menethoran** [dahlb/ha_kia_hyundai#114] [General]
Just chiming in to confirm that this integration DOES work with the 2024 Telluride. Not sure if theres tracking for verified working units but i wanted to chime in with the rebrand of the app, it looks like backend is all identical.

**Comment by Jyates18** [comment]
Did you get remote start and other functions? All I'm seeing is door locks as a switch on my 2023 in home assistant, everything else is sensors and binary sensors

**Comment by Menethoran** [comment]
No, unlock and monitoring are the only features with direct access. I'm
poking around trying to see what the API call for remote start is.

  - Robert Burton Thompson V
    President: RnD Technologies
    Ph: (717)466-9069
